concordium-consensus/tests/globalstate/GlobalStateTests/AccountsMigrationP6ToP7.hs

```diff
@@ -100,7 +100,8 @@ testAccount persisting stake =
           _accountEncryptedAmount = dummyAccountEncryptedAmount,
           _accountReleaseSchedule = Transient.emptyAccountReleaseSchedule,
           _accountStaking = stake,
-          _accountStakeCooldown = Transient.emptyCooldownQueue (accountVersion @av)
+          _accountStakeCooldown = Transient.emptyCooldownQueue (accountVersion @av),
+          _accountTokenStateTable = CFalse
```

Comment:
This could be adapted to also support PLTs.

Reply:
Shouldn't it always be `CFalse` for P7?